<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eduversum</class>
 <widget class="QWidget" name="Eduversum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eduversum</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="applicationsButton">
        <property name="styleSheet">
         <string>border:0</string>
        </property>
        <property name="text">
         <string>Applications</string>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="webButton">
        <property name="styleSheet">
         <string>border:0</string>
        </property>
        <property name="text">
         <string>Weblinks</string>
        </property>
        <property name="icon">
         <iconset resource="../eduversum.qrc">
          <normaloff>:/res/web.svg</normaloff>:/res/web.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="docsButton">
        <property name="styleSheet">
         <string>border:0</string>
        </property>
        <property name="text">
         <string>Teaching Aids</string>
        </property>
        <property name="icon">
         <iconset resource="../eduversum.qrc">
          <normaloff>:/res/docs.svg</normaloff>:/res/docs.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="helpButton">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>border:0</string>
        </property>
        <property name="text">
         <string>Help</string>
        </property>
        <property name="icon">
         <iconset resource="../eduversum.qrc">
          <normaloff>:/res/help.svg</normaloff>:/res/help.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="contentStackedWidget">
     <property name="styleSheet">
      <string>margin:0</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="appsPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="leftwidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTreeWidget" name="categoriesTreeWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>3332</height>
              </size>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string>1</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="content" native="true">
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="topWidget" native="true">
              <layout class="QGridLayout" name="gridLayout_6">
               <property name="margin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QFrame" name="frame0">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Search:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="searchLineEdit">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <spacer name="horizontalSpacer2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>168</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="0" column="3">
                   <widget class="QComboBox" name="comboBox">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <item>
                     <property name="text">
                      <string>All Programs</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Only installed</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Only not installed</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QTreeWidget" name="treeWidget">
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="rootIsDecorated">
                  <bool>false</bool>
                 </property>
                 <property name="headerHidden">
                  <bool>true</bool>
                 </property>
                 <column>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QFrame" name="frame">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_7">
                  <property name="margin">
                   <number>0</number>
                  </property>
                  <item row="0" column="1">
                   <widget class="QPushButton" name="execPushButton">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Start</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../eduversum.qrc">
                      <normaloff>:/res/exec.png</normaloff>:/res/exec.png</iconset>
                    </property>
                    <property name="checkable">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="4">
                   <widget class="QPushButton" name="installRemovePushButton">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Install</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../eduversum.qrc">
                      <normaloff>:/res/add.png</normaloff>:/res/add.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QPushButton" name="examplePushButton">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>Example Files</string>
                    </property>
                    <property name="icon">
                     <iconset resource="../eduversum.qrc">
                      <normaloff>:/res/example.png</normaloff>:/res/example.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3">
                   <spacer name="horizontalSpacer3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>175</width>
                      <height>24</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="bottomWidget" native="true">
              <layout class="QGridLayout" name="gridLayout_9">
               <property name="margin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QTextBrowser" name="descriptionTextBrowser">
                 <property name="openExternalLinks">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="webPage">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTreeWidget" name="webTreeWidget">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
         </widget>
         <widget class="QTextBrowser" name="webTextBrowser">
          <property name="styleSheet">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aidsPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTreeWidget" name="docsTreeWidget">
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true"/>
           </property>
          </column>
         </widget>
         <widget class="QTextBrowser" name="docsTextBrowser"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionUeberEduversum">
   <property name="text">
    <string>About Eduversum</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionEduversumHilfe">
   <property name="text">
    <string>Eduversum Help</string>
   </property>
  </action>
  <action name="actionDisclaimer">
   <property name="text">
    <string>Disclaimer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../eduversum.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionEduversumHilfe</sender>
   <signal>activated()</signal>
   <receiver>Eduversum</receiver>
   <slot>showHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUeberEduversum</sender>
   <signal>activated()</signal>
   <receiver>Eduversum</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applicationsButton</sender>
   <signal>clicked()</signal>
   <receiver>Eduversum</receiver>
   <slot>showAppsPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>docsButton</sender>
   <signal>clicked()</signal>
   <receiver>Eduversum</receiver>
   <slot>showDocsPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>docsTreeWidget</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>Eduversum</receiver>
   <slot>showDoc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webTreeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>Eduversum</receiver>
   <slot>showWebDescription()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webTreeWidget</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>Eduversum</receiver>
   <slot>showWeblink()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webButton</sender>
   <signal>clicked()</signal>
   <receiver>Eduversum</receiver>
   <slot>showWeblinkPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>docsTreeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>Eduversum</receiver>
   <slot>showDocDescription()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showHelp()</slot>
  <slot>showAbout()</slot>
  <slot>showStuff()</slot>
  <slot>showOpenSource()</slot>
  <slot>showSeminarixLatex()</slot>
  <slot>showAppsPage()</slot>
  <slot>showDocsPage()</slot>
  <slot>showDoc()</slot>
  <slot>showWebDescription()</slot>
  <slot>showWeblink()</slot>
  <slot>showWeblinkPage()</slot>
  <slot>showDocDescription()</slot>
 </slots>
</ui>
